### ISSUE TITLE
Errors in master

### ISSUE THREAD

**Issue by itziakos** [simphony/simphony-common#194]
Due to merging multiple PRs at the same time some test failures have passed through, sorry about that.

```python
======================================================================
ERROR: test_compare_mesh_datasets_equal (simphony.testing.tests.test_utils.TestCompareMeshDatasets)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "simphony/testing/tests/test_utils.py", line 123, in test_compare_mesh_datasets_equal
    mesh.add_point(point)
AttributeError: 'Mesh' object has no attribute 'add_point'
======================================================================
ERROR: test_compare_mesh_datasets_not_equal (simphony.testing.tests.test_utils.TestCompareMeshDatasets)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "simphony/testing/tests/test_utils.py", line 154, in test_compare_mesh_datasets_not_equal
    mesh.add_point(point)
AttributeError: 'Mesh' object has no attribute 'add_point'
----------------------------------------------------------------------
Ran 509 tests in 101.489s
```

**Comment by roigcarlo** [comment]
@itziakos I will adress this after #164

**Comment by roigcarlo** [comment]
Fixed in #177 177
